corehq/apps/data_interfaces/utils.py

```diff
@@ -34,3 +35,33 @@ def add_cases_to_case_group(domain, case_group_id, uploaded_data):
         case_group.save()
 
     return response
+
+
+def archive_forms(user, uploaded_data):
```

Comment:
I'm not 100% sure where it would go, but can we limit the number of forms you're allowed to update somewhere? I just worry that this could create a huge indexing load on couch if someone tries to archive 100,000 forms at once.